sys/Kconfig

```diff
@@ -58,6 +58,16 @@ rsource "usb/Kconfig"
 rsource "xtimer/Kconfig"
 rsource "ztimer/Kconfig"
 
+config MODULE_CPP
+    bool "Use CPP compiler"
+    depends on TEST_KCONFIG
+    depends on HAS_CPP
+
+config MODULE_LIBSTDCPP
```

Comment:
I think this should depend on `HAS_CPP` and `select MODULE_CPP`.

Reply:
Done